### DISCUSSION TITLE
nuxt generates double stylesheet links

### DISCUSSION THREAD

**Discussion by fahdaddi** [nuxt/nuxt#9330] [Questions (Nuxt 2)]
hello, I'm working on a nuxt/oruga/talwind project. when I import scss file from a .vue page it got rendered twice in the page.

ex:
**pages/index.vue**
```
<template>
  <div class="wrap">
    <Usp/> /*custom component*/
  </div>
</template>
...

<style lang="scss">
@import "~/assets/css/pages/home";
</style>
```

**assets/css/pages/home.scss**
```
@import "../_variables";
ul.usp {
  padding: $padding-xs 0;

  > li {
    display: flex;
    flex-wrap: nowrap;
    color: $color-secondary;
    white-space: nowrap;
  }
}
```

when I inspect the ul.usp in dom I get this:
<img width="325" alt="Capture d’écran 2021-05-24 à 15 13 59" src="https://user-images.githubusercontent.com/23249370/119385628-473e3780-bcbe-11eb-9fbd-707748bdb83f.png">

**this happens  because the css is called twice (in head + body )**
<div style="display= flex">
<img width="300" alt="Capture d’écran 2021-05-24 à 15 14 18" src="https://user-images.githubusercontent.com/23249370/119385660-5ae99e00-bcbe-11eb-9a3f-735a3d8df1bc.png">
<img width="300" alt="Capture d’écran 2021-05-24 à 15 14 59" src="https://user-images.githubusercontent.com/23249370/119385666-5de48e80-bcbe-11eb-96ad-239701db9263.png">
</div>

**Comment by danielroe** [comment]
@fahdaddi This is [a current issue in **extract-css-chunks-webpack-plugin**](https://github.com/nuxt/nuxt.js/issues/8060#issuecomment-756750084) with a [pending fix](https://github.com/faceyspacey/extract-css-chunks-webpack-plugin/pull/307).
